rest/handler.go

```diff
@@ -338,7 +338,20 @@ func (h *handler) validateAndWriteHeaders(method handlerMethod, accessPermission
 	// look up the database context:
 	if keyspaceDb != "" {
 		var err error
-		if dbContext, err = h.server.GetActiveDatabase(keyspaceDb); err != nil {
+		dbContext, err = h.server.GetActiveDatabase(keyspaceDb)
```

Comment:
If we don't let the database load, I don't understand why we need a dbcontext for PUT and POST - fixing this would require a PUT to override it but I don't think it should fail?

Maybe test these in separate functions?